docs/tla/README.md

```diff
@@ -0,0 +1,18 @@
+This directory contains TLA+ models for specific Client behavior. Each
+`$MODEL.tla` is documented:
+
+- in its accompanying configuration `$MODEL.cfg`;
+- with a preamble, above the (e.g.) `---- MODULE Reply ----` line;
+- inline; and
+- with any footnotes or concluding commentary below the `====` line.
+
+With `$TLA_BIN` set in your path, you can run `make $MODEL` to run the model and
+render its state graph. (Or you can let the [TLA+ extension] for Visual Studio
+Code/Codium do all this for you.)
```

Comment:
I tried the $TLA_BIN patheway - here are the specific downloads I used:
1. Downloaded the latest `tla2tools.jar` from https://github.com/tlaplus/tlaplus/releases
2. exported `TLA_BIN` as "java -jar /full/path/to/tla2tools.jar"
3. Run `make Reply`

This worked and successfully generated the `Reply.png`. :tada:  If these are the correct steps for the non-extension pathway, that might be worth explicitly adding here.

Reply:
Good idea.  Documented inline in 5875a0c059db1400f094b5231d9341889af5b7f7.